README.md

```diff
@@ -54,7 +54,12 @@ HCL Language Support: https://plugins.jetbrains.com/plugin/7808-hashicorp-terraf
 
 2. In command terminal navigate to the folder where the project has been cloned to
 
-3. Create Terraform workspaces corresponding to the TDR environments:
+3. Clone Terraform modules repository
+   ```
+   [location of project] $ git clone git@github.com:nationalarchives/tdr-terraform-environments.git
```

Comment:
Should this be tdr-terraform-modules rather than tdr-terraform-environments?

As discussed in person, it would be nice if we didn't have to clone tdr-terraform-modules into every Terraform project, but we can't use Terraform variables to get round this because [Terraform doesn't allow you to use variables in the module source parameter](https://github.com/hashicorp/terraform/issues/1439).

Reply:
Thanks, corrected